docs/install/Knative-with-OpenShift.md

```diff
@@ -5,219 +5,50 @@ weight: 15
-rerun the command to see the current status.
+* An OpenShift 4 cluster is required for installation. Visit [try.openshift.com](try.openshift.com) for information on setting up a cluster. You will need cluster administrator privileges to install and use Knative on an OpenShift cluster. 
 
-> Note: Instead of rerunning the command, you can add `--watch` to the above
-> command to view the component's status updates in real time. Use CTRL+C to
-> exit watch mode.
+> Note: Currently, long-running clusters are not supported on OpenShift 4.
 
-Set `priviledged` to `true` for the `istio-sidecar-injector`:
 
-```shell
-oc get cm istio-sidecar-injector -n istio-system -oyaml  \
-| sed -e 's/securityContext:/securityContext:\\n      privileged: true/' \
-| oc replace -f -
-```
+## Installing the Knative Serving Operator
 
-Restart the `sidecar-injector` pod if `SELinux` is enabled:
+1. Go to **Catalog > OperatorHub** in the OpenShift Web Console. A list of operators for OpenShift, provided by Red Hat as well as a community of partners and open-source projects is provided. Click on the **Knative Serving Operator** tile.
 
-```shell
-if getenforce | grep -q Disabled
-then
-    echo "SELinux is disabled, no need to restart the pod"
-else
-    echo "SELinux is enabled, restarting sidecar-injector pod"
-    oc delete pod -n istio-system -l istio=sidecar-injector
-fi
-```
+> **NOTE:** Use the **Filter by Keyword** box to facilitate the search of the Knative Serving operator in the catalog.
 
-<!-- TODO: need instruction about installing Cert-Manager with `oc`-->
+![KSO Tile](images/knative_serving_tile_highlighted.png)
 
-## Installing Knative Serving
+2. A **Show Community Operator** dialog box will open. Click **Continue** to proceed.
 
-Next, we'll install [Knative Serving](https://github.com/knative/serving).
+3. The **Knative Serving Operator** descriptor screen will appear. Click **Install**.
 
-First, run the following to grant the necessary privileges to the service
-accounts istio will use:
+![KSO Install Screen](images/knative_serving_operator_screen.png)
 
-```shell
-oc adm policy add-scc-to-user anyuid -z build-controller -n knative-build
-oc adm policy add-scc-to-user anyuid -z controller -n knative-serving
-oc adm policy add-scc-to-user anyuid -z autoscaler -n knative-serving
-oc adm policy add-scc-to-user anyuid -z kube-state-metrics -n knative-monitoring
-oc adm policy add-scc-to-user anyuid -z node-exporter -n knative-monitoring
-oc adm policy add-scc-to-user anyuid -z prometheus-system -n knative-monitoring
-oc adm policy add-cluster-role-to-user cluster-admin -z build-controller -n knative-build
-oc adm policy add-cluster-role-to-user cluster-admin -z controller -n knative-serving
-```
+4. On the **Create the Operator Subscription** screen, create a new subscription by cliking on the **Subscribe** button. This will install the Knative Serving Operator in the project `openshift-operators` and Knative Serving in the `knative-serving` project.
```

Comment:
```suggestion
4. On the **Create the Operator Subscription** screen, create a new subscription by clicking on the **Subscribe** button. This will install the Knative Serving Operator in the project `openshift-operators` and Knative Serving in the `knative-serving` project.
```